crates/holochain/tests/tests/sharded_gossip/mod.rs

```diff
@@ -847,1052 +847,3 @@ async fn fullsync_sharded_local_gossip() -> anyhow::Result<()> {
-    let mock_network: AdapterFactory = Arc::new(mock_network);
-
-    // Setup the bootstrap.
-    let (bootstrap, _shutdown) = run_bootstrap(data.agent_to_info.values().cloned()).await;
-    // Setup the network.
-    let mut tuning = KitsuneP2pTuningParams::default();
-    tuning.gossip_strategy = "sharded-gossip".to_string();
-    tuning.gossip_dynamic_arcs = true;
-
-    let mut network = KitsuneP2pConfig::empty();
-    network.bootstrap_service = Some(bootstrap);
-    network.transport_pool = vec![TransportConfig::Mock {
-        mock_network: mock_network.into(),
-    }];
-    network.tuning_params = Arc::new(tuning);
-    let mut config = ConductorConfig::empty();
-    config.network = network;
-
-    // Add it to the conductor builder.
-    let builder = ConductorBuilder::new().config(config);
-    let mut conductor = SweetConductor::from_builder(builder).await;
-
-    // Install the real agent alice.
-    let apps = conductor.setup_app("app", [&dna_file]).await.unwrap();
-
-    let (alice,) = apps.into_tuple();
-    let alice_p2p_agents_db = conductor.get_p2p_db(alice.cell_id().dna_hash());
-    let alice_kit = alice.agent_pubkey().to_kitsune();
-
-    // Spawn a task to update alice's agent info.
-    tokio::spawn({
-        let alice_info = alice_info.clone();
-        async move {
-            loop {
-                let info = alice_p2p_agents_db.p2p_get_agent(&alice_kit).await.unwrap();
-
-                {
-                    if let Some(info) = &info {
-                        eprintln!("Alice coverage {:.2}", info.storage_arc().coverage());
-                    }
-                    *alice_info.lock() = info;
-                }
-
-                tokio::time::sleep(std::time::Duration::from_secs(5)).await;
-            }
-        }
-    });
-
-    let num_actions = data.ops.len();
-    loop {
-        let mut count = 0;
-
-        for hash in data
-            .ops
-            .values()
-            .map(|op| ActionHash::with_data_sync(&op.action()))
-        {
-            let record: Option<Record> = conductor.call(&alice.zome("zome1"), "read", hash).await;
-            if record.is_some() {
-                count += 1;
-            }
-            // let gets = num_gets.load(std::sync::atomic::Ordering::Relaxed);
-            // let misses = num_misses.load(std::sync::atomic::Ordering::Relaxed);
-            // eprintln!(
-            //     "checked {:.2}%, got {:.2}%, missed {:.2}%",
-            //     i as f64 / num_actions as f64 * 100.0,
-            //     count as f64 / num_actions as f64 * 100.0,
-            //     misses as f64 / gets as f64 * 100.0
-            // );
-        }
-        eprintln!(
-            "DONE got {:.2}%, {} out of {}",
-            count as f64 / num_actions as f64 * 100.0,
-            count,
-            num_actions
-        );
-
-        tokio::time::sleep(std::time::Duration::from_secs(10)).await;
-    }
-}
-
-*/
```

Comment:
@neonphog are these tests still useful?

Reply:
I'd say this is the wrong abstraction layer to be writing these tests. If this test is about validating the gossip round initiation and resultant gets/publishes, the mock peers should be abstracted at that level.

Mocking at the network layer creates a brittle encapsulation breakage where if you change the networking at all, you'd have to go update the nitty-gritty of this test when you're not in the mindset of it, as we've seen happen here.